### ISSUE TITLE
Fundamental Implementations for the faster-rcnn

### ISSUE THREAD

**Issue by andersy005** [andersy005/keras-faster-rcnn#1]
- [x] Fixed Batch Normalization
- [x] Roi Pooling Conv
- [ ] Roi Helpers
- [ ] config
- [x] Data Augment
- [ ] Data Generators
- [ ] Losses
- [x] pascal_voc_parser
- [x] Resnet
- [x] VGG

**Comment by themechanicalcoder** [comment]
@andersy005  can you please tell how to train the model I was going through the code but could not find how to train the model
